src/app/business/routes/dataset/sagas/index.js

```diff
@@ -78,14 +63,7 @@ function* fetchItem({payload}) {
 
 function* fetchPersistent(request) {
     const state = yield select();
-    let jwt;
-
-    if (typeof window !== 'undefined') {
-        const cookies = cookie.parse(window.document.cookie);
-        if (cookies['header.payload']) {
-            jwt = cookies['header.payload'];
-        }
-    }
+    const jwt = getJWTFromCookie();
```

Comment:
This looks like an incomplete refactoring, but maybe there is a reason you're not using the `jwt = tryRefreshToken(actions.persistent.failure)` pattern here.

Reply:
I don't know what to do here, because this function come from the rebase...

Reply:
It's your choice really : either update it here with the new pattern or add a small issue asking for the refactoring of these. I have no preference, but if the latter allows for this to be merged faster then it's for the best !

Reply:
I can add the refactoring on this but wanted your confirmation it can apply here first

Reply:
👍